x-pack/plugins/enterprise_search/public/applications/shared/tables/inline_editable_table/types.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { Column } from '../reorderable_table/types';
+
+export interface FormErrors {
+  [key: string]: string | undefined;
```

Comment:
I wonder if this needs the `| undefined`.  This should be equivalent to 

```
export interface FormErrors {
  [key: string]: string;
}
```

I think?

Reply:
Good point